src/main/java/Model/Utilities/Moves.java

```diff
@@ -13,92 +15,51 @@
     /**
      * Choosing the correct function to call
      * 
-     * @param position
-     * @param isWhite
-     * @param board
-     * @param pieceType
-     * @return list of possible moves
+     * @param position of the piece we are trying to move
+     * @param game that the piece is playing in
+     * @return list of possible moves including Illegal Moves
      */
-    public static ArrayList<Position> chooseMove(Position position, boolean isWhite, Game board, PieceType pieceType) {
+    public static ArrayList<Position> getAllMovesIncludingIllegalMoves(Position position, Game game) {
```

Comment:
i think you have fallen a bit too in love with static methods here, and think you are living dangerously close to mutating your game state accidentally. not such a big deal here, and works perfectly, but something to think about